event_image/models/event.py

```diff
@@ -0,0 +1,11 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class EventWithImage(models.Model):
+
+    _inherit = 'event.event'
+
+    image = fields.Binary('Image', attachment=True)
```

Comment:
Even though this technically works, I'd do something like this here:

fields:
https://github.com/odoo/odoo/blob/2fcc8a717e9e399692dfaa2d68dd0fcead825de5/addons/product/models/product_template.py#L158-L171

create:
https://github.com/odoo/odoo/blob/2fcc8a717e9e399692dfaa2d68dd0fcead825de5/addons/product/models/product_template.py#L347-L348

write:
https://github.com/odoo/odoo/blob/2fcc8a717e9e399692dfaa2d68dd0fcead825de5/addons/product/models/product_template.py#L376